### ISSUE TITLE
Multiple model merger

### ISSUE THREAD

**Issue by hariharan1412** [colmap/colmap#2265]
After I ran colmap mapper, I got 5 sparse folders as output. The poses in sparse folders 0 and 4 are good, but sparse folders 1, 2, and 3 are bad, so is there a way to merge sparse folders 0 and 4?

**Comment by tsattler** [comment]
You can try the `model_merger` from the command line. It should work as long as there are enough images that are contained in both models.

**Comment by hariharan1412** [comment]
Yes, that's the problem. I know there are no matching images between them, so i am asking is there a way I can edit or use other tools to merge the model?

**Comment by tsattler** [comment]
Do you have poses in any common coordinate frame, e.g., GPS? In this case, I could see a way. Otherwise, without any overlap between both models, I don't see how you could get any alignment.

**Comment by hariharan1412** [comment]
yes i have, I have used Spatial matcher only to match the images. Any lead will be helpfull

**Comment by tsattler** [comment]
You  can try the exhaustive matcher (or something that is more exhaustive) to see whether this connects the models. Otherwise, you could use the `model_aligner` to align the individual models to GPS coordinates. You could then try to merge both reconstructions (but Colmap does not provide code for this if the individual models do not overlap).

**Comment by ahojnnes** [comment]
Closing: model_merger needs overlapping images; alternatives suggested.
